test/parallel/xyk-pallet.multiswap.errors.test.ts

```diff
@@ -81,18 +78,15 @@ describe("Multiswap - error cases: disabled tokens", () => {
         tokenIds[tokenIds.length - 1],
         testUser1,
       );
-      const multiSwapOutput = await multiSwapBuy(
+      const multiSwapOutput = await multiSwapBuyMarket(
         testUser1,
         tokenIds,
         new BN(1000),
         BN_TEN_THOUSAND,
       );
 
-      const eventResponse = getEventResultFromMangataTx(multiSwapOutput, [
-        "xyk",
-        "MultiSwapAssetFailedOnAtomicSwap",
-      ]);
-      expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+      const eventResponse = getEventResultFromMangataTx(multiSwapOutput);
+      expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicFailed);
```

Comment:
you forgot to evaluate the error message, plz! :pray:

Reply:
Added